components/aws/sagemaker/batch_transform/README.md

```diff
@@ -0,0 +1,98 @@
+# SageMaker Batch Transform Kubeflow Pipeline component
+## Summary
+Component to get inferences for an entire dataset in Sagemaker from a Kubeflow Pipelines workflow.
```

Comment:
`to get inferences` sounds a bit off. Perhaps `Component to produce inference outputs for ...`?

Also any reference to SageMaker should capitalise both the S and the M.

Reply:
good catch on SageMaker. I will leave the summary portion as is since I picked it from the AWS documentation

Reply:
nit: Sagemaker -> SageMaker.

Reply:
updated